### PR TITLE
chore(Messaggi a valore legale): [IAMVL-19] Display legal message attachments

### PR DESCRIPTION
**Pull request by fabriziofff**
## Short description
This pr implements the `MvlAttachments` component, in order to render the attachments of a legal message.
This pr **doesn't** implement the download of an attachment https://pagopa.atlassian.net/browse/IAMVL-27 and the serialization of the user preferences about the download https://pagopa.atlassian.net/browse/IAMVL-19

No Attachment            |  At least one attachment
:-------------------------:|:-------------------------:
Schermata 2021-12-16 alle 11 28 41 |  Schermata 2021-12-16 alle 11 07 41

https://user-images.githubusercontent.com/26501317/146351651-cbc338dd-67ee-48d2-8726-e7614ff58f08.mov

## List of changes proposed in this pull request
- Implemented `MvlAttachments`in order to render the attachments of a legal message
- Added `DownloadAttachmentConfirmationBottomSheet` in order to ask confirmation for the download of an attachment
- Added  `MvlAttachmentId` in order to update the `MvlAttachment` model that now incluAdes an unique id
- Added `formatByte` function in order to format the bytes

## How to test
- Execute `    NavigationService.dispatchNavigationAction(
      navigateToMvlDetailsScreen({ id: "id" as UIMessageId })
    )`

### CODE REVIEW

**Review by pietro909 — APPROVED** [review]

**Comment by pietro909** [comment]
Approved from my side. @fabriziofff are we waiting for @thisisjp 's approval?

**Review by thisisjp — APPROVED** [review]

**Pull request merged**
